README.md

```diff
@@ -25,21 +25,21 @@ Jarbas is in charge of making data from [CEAP](https://github.com/datasciencebr/
 
 ### Reimbursement
 
-Each `Reimbursement` object is a reimbursement claim made by a congressperson. Each reimbursement isidentified by an unique combination of `year`, `applicant_id` and `document_id`.
+Each `Reimbursement` object is a reimbursement claimed by a congressperson and identified publicly by its `document_id`.
 
 #### Retrieving a specific reimbursement
 
-##### `GET /api/reimbursement/<year>/<applicant_id>/<document_id>/`
+##### `GET /api/reimbursement/<document_id>/`
 
-Details from a specific reimbursement. If `receipt_url` wasn't fecthed yet, the server **won't** try to fetche it.
+Details from a specific reimbursement. If `receipt_url` wasn't fetched yet, the server **won't** try to fetche it automatically.
```

Comment:
lol you corrected a typo I did not see, `try to fetche it` -> `try to fetch it` there is an extra `e` there =P